pudb/debugger.py

```diff
@@ -546,6 +546,8 @@ def get_lines(self, debugger_ui):
             return [SourceLine(debugger_ui, self.file_name)]
 
         breakpoints = debugger_ui.debugger.get_file_breaks(self.file_name)[:]
+        breakpoints = [lineno for lineno in breakpoints if
```

Comment:
I think this line is unnecessary. It might be necessary if we ever save the "enabled" state to the breakpoints file, however.